core/src/ledger_cleanup_service.rs

```diff
@@ -15,7 +15,7 @@ use std::time::Duration;
 
 // - To try and keep the RocksDB size under 400GB:
 //   Seeing about 1600b/shred, using 2000b/shred for margin, so 200m shreds can be stored in 400gb.
-//   at 5k shreds/slot at 50k tps, this is 500k slots (~5 hours).
+//   at 5k shreds/slot at 50k tps, this is 500k slots (~55 hours).
```

Comment:
This line tripped me up as well; however, my correction is held up as part of a larger PR so let's come to a consensus here and I'll rebase :).

I think the time value of (roughly) 5 hours is correct, but the slot value (500k) is incorrect.
```
400 GB = 400,000,000,000 bytes * (1 shred / 2000 bytes) = 200,000,000 shreds // current comment agrees
200,000,000 shreds * (1 slot / 5,000 shreds) = 40,000 slots @ 50k TPS load
40,000 slots * 0.4 sec / slot * (1 min / 60 sec) * (1 hour / 60 min) = 4.44... hours
```

Reply:
Ohhhh, this makes so much more sense. I just trusted the slots, and didn't go further back.
I agree with your math and will fix up.

Reply:
@steviez , pushed. Please take a look